src/openforms/forms/models/form.py

```diff
@@ -244,7 +244,10 @@ class Meta:
         verbose_name_plural = _("forms")
 
     def __str__(self):
-        return self.admin_name
+        if self._is_deleted:
+            return "{}{}".format(self.admin_name, _(" (deleted)"))
```

Comment:
Always put as much as possible in the translatable string so that arguments can be re-ordered in the translations.

```suggestion
            return _("{name} (deleted)").format(name=self.admin_name)
```